config-template.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copy this file to config.js and fill it in with your own information.
+ */
+module.exports = {
```

Comment:
Sorry, I mean specifically where the authentication token comes from in GitHub, etc.